File: examples/mvp-java/src/main/java/co/windly/limbosample/presentation/main/root/settings/SettingsModule.java
package co.windly.limbosample.presentation.main.root.settings;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class SettingsModule {

  //region Contribution

  @ContributesAndroidInjector
  abstract SettingsFragment contibuteAndroidInjector();

  //endregion
}

File: examples/mvp-kotlin/src/main/java/co/windly/limbosample/presentation/main/root/settings/SettingsModule.kt
package co.windly.limbosample.presentation.main.root.settings

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class SettingsModule {

  //region Contribution

  @ContributesAndroidInjector
  abstract fun contributeAndroidInjector(): SettingsFragment

  //endregion
}
